### ISSUE TITLE
Set up poetry

### ISSUE THREAD

**Issue by Alicegaz** [eora-ai/torchok#61]
It would be great to have dependencies managed by poetry.

**Comment by VladVin** [comment]
Yes, we now have `poetry` setup. The only thing left to do is to give instructions to contributors on how to use it. So, I keep this issue open until this is resolved
